client/src/resources/operation/test.rs

```diff
@@ -14,12 +14,12 @@ mod errors_on_missing_fields_for_type {
             fn $type_to_check() {
                 let json = format!(
                     r#"{{
-                        "id":"1",
-                        "paging_token":"7",
-                        "type_i":{},
-                        "type":"{}",
-                        "transaction_hash":"123"
-                    }}"#,
+                                    "id":"1",
```

Comment:
Looks like you are on an old version of rust now, try updating so this doesn't change.

Reply:
oh man... let me check.

Reply:
@kbacha just ran cargo fmt --all w/ 1.26.2 installed and nothing changed. However, moved it back and ran fmt and still nothing changed 🤔 . Anyway just reformatted and pushed.